src/analytics/create-step-tracker.ts

```diff
@@ -20,11 +20,6 @@ import StepTracker from './step-tracker';
  * stepTracker.trackCheckoutStarted();
  * ```
  *
- * @alpha
- * Please note that `StepTracker` is currently in an early stage
- * of development. Therefore the API is unstable and not ready for public
```

Comment:
I believe it was mixed with the previous commit during the squash, right?

Reply:
Yes, @andriiVitvitskyi1990 can you please fix this. Otherwise, the PR looks good to me 🙂

Reply:
Fixed